### PR TITLE
Proposal: Standardising live blog teasers labels (and hierarchy).

### PR DESCRIPTION
**Pull request by josh-wade-ft**
Design proposal for live teaser labels

### CODE REVIEW

File: proposals/accepted/0000-live-teaser-labels.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# put your proposal title here
+
+> Standardising live blog teasers labels (and hierachy).
+
```
Comment (chee) [inline proposals/accepted/0000-live-teaser-labels.md:4]:
```suggestion
# Standardising live blog teasers labels (and hierarchy).

```

**Comment by chee** [comment]
here's a link to the rendered proposal: https://github.com/josh-wade-ft/origami/blob/patch-3/proposals/accepted/0000-live-teaser-labels.md

**Comment by JakeChampion** [comment]
[In Slack](https://financialtimes.slack.com/archives/C01481FKWA2/p1615380421110900?thread_ts=1615380379.110500&cid=C01481FKWA2) Luke G and David both said they like this proposal 👍

**Comment by josh-wade-ft** [comment]
@JakeChampion @chee From Origami's perspective; how many designers would you need that sort of soft approval from? (as Luke and Dave have done)

This could be instance to define that ^ :)

**Comment by chee** [comment]
> @JakeChampion @chee From Origami's perspective; how many designers would you need that sort of soft approval from? (as Luke and Dave have done)
> 
> This could be instance to define that ^ :)

That's a good point. It seems to me like something that should be formalized by design, so there are not conflicting recommendations being made to Origami.

**Comment by josh-wade-ft** [comment]
> > @JakeChampion @chee From Origami's perspective; how many designers would you need that sort of soft approval from? (as Luke and Dave have done)
> > This could be instance to define that ^ :)
> 
> That's a good point. It seems to me like something that should be formalized by design, so there are not conflicting recommendations being made to Origami.

Hey! Yeah that makes sense - right now, the only 3 volunteers of design-ops is myself, Luke and Xuan 😄 But I can formalise with the owners of the homepage. Wdyt?

**Comment by chee** [comment]
> > > @JakeChampion @chee From Origami's perspective; how many designers would you need that sort of soft approval from? (as Luke and Dave have done)
> > > This could be instance to define that ^ :)
> > 
> > 
> > That's a good point. It seems to me like something that should be formalized by design, so there are not conflicting recommendations being made to Origami.
> 
> Hey! Yeah that makes sense - right now, the only 3 volunteers of design-ops is myself, Luke and Xuan :smile: But I can formalise with the owners of the homepage. Wdyt?

That sounds great. If we have a formalized structure for how design makes a recommendation to Origami, we can have somewhere to point to when we want to understand how a decision was made when it's time to revisit it.

**Comment by josh-wade-ft** [comment]
> > > > @JakeChampion @chee From Origami's perspective; how many designers would you need that sort of soft approval from? (as Luke and Dave have done)
> > > > This could be instance to define that ^ :)
> > > 
> > > 
> > > That's a good point. It seems to me like something that should be formalized by design, so there are not conflicting recommendations being made to Origami.
> > 
> > 
> > Hey! Yeah that makes sense - right now, the only 3 volunteers of design-ops is myself, Luke and Xuan 😄 But I can formalise with the owners of the homepage. Wdyt?
> 
> That sounds great. If we have a formalized structure for how design makes a recommendation to Origami, we can have somewhere to point to when we want to understand how a decision was made when it's time to revisit it.

@chee Yeah! I think we can formalise publicly in design-ops channel, I'll put a slide together - but for this specific proposal there are no objections from design, I'll get an editorial stakeholder to have a look over and give their consent just to be safe, as it's the homepage.

**Comment by josh-wade-ft** [comment]
@chee @JakeChampion This has been blocked by editorial :(

**Comment by notlee** [comment]
As this work was blocked by editorial and Josh has left (😢) I'm going to close this PR

**Pull request closed without merging**